velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -79,19 +87,41 @@ void HiveConnectorTestBase::writeToFile(
     const std::vector<RowVectorPtr>& vectors,
     std::shared_ptr<dwrf::Config> config,
```

Comment:
We want to drop the dependency on dwrf, this means we don't want to see the dwrf namesapce being hard coded here. The desired interface is something like this:
```
void writeToFile(
      const std::string& filePath,
      const std::vector<RowVectorPtr>& vectors,
      const TypePtr& schema,
      std::shared_ptr<dwio::common::WriterOptions> options);
```
This allows the user be able to supply custom WriterOptions. the dwrf::WriterOptions contains the dwrf::Config but Parquet doesn't have a counterpart parquet::Config. But they all have WriterOptions and we can unify the interface.